src/test/java/core/basesyntax/JavaStreamApiTest.java

```diff
@@ -0,0 +1,121 @@
+package core.basesyntax;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.NoSuchElementException;
+
+public class JavaStreamApiTest {
+    private List<Integer> numbers;
+    private List<String> elements;
+    private List<Integer> digits;
+    private List<People> peopleList;
+    private JavaStreamApi javaStreamApi;
+
+    @Before
+    public void setUp() {
+        numbers = Arrays.asList(1, 2, 3, 4, 5);
+        elements = Arrays.asList("a1", "a2", "a3", "a1");
+        digits = Arrays.asList(6, 2, 3, 7, 2, 5);
+        peopleList =
+                Arrays.asList(new People("Вася", 16, People.Sex.MAN),
+                        new People("Петя", 23, People.Sex.MAN),
+                        new People("Елена", 42, People.Sex.WOMEN,
+                                Arrays.asList(new Cat("Матроскин", 2),
+                                        new Cat("Барсик", 3))),
+                        new People("Иван Иванович", 69, People.Sex.MAN));
+        javaStreamApi = new JavaStreamApi();
+    }
+
+    @Test
+    public void oddSum() {
+        Integer expectedResult = 9;
+        Integer actualResult = javaStreamApi.oddSum(numbers);
+        Assert.assertEquals(expected, result);
```

Comment:
```suggestion
        Assert.assertEquals(expectedResult, actualResult);
```
краще змінити назви змінних у всіх методів